### DISCUSSION TITLE
Forks list should be more informative adding functions like star count, last commit time, description or a like count.

### DISCUSSION THREAD

**Discussion by cicadabear** [community/community#35587] [Repositories]
For example, a Gnome extension [system-monitor](https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet) is lacking maintenance for a while so it doesn't support gnome 42, but someone is keeping the maintaining job in the fork project [system-monitor-next](https://github.com/mgalgs/gnome-shell-system-monitor-applet).    

If the GitHub fork list has something like sort by the last commit time or a brief description, there would be no time wasted on finding an alternative.   

And an informative fork list is not only a benefit to unmaintained projects, but there are also a lot of fork projects getting more attention than the original one or having some niche features, and it should be known by more people.   

We may need some functions like:     

- star count
- last commit time
- brief description 
- sort by star count or last commit time  
- or maybe a like count

**Comment by ghost** [comment]
I was just thinking about this today.  I've been experimenting with a gem for my project, and while it appears to do everything I need, I'd be curious _**why**_ some of the other forks were created and what makes a particular fork different/special/better.  I feel having that information available at a glance would be incredibly useful.

**Comment by ghost** [comment]
@cicadabear Might be worth keeping an eye on this: https://github.com/orgs/community/discussions/40968
